code/src/index.css

```diff
@@ -11,3 +11,128 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+header {
```

Comment:
As a general suggestion, look over your spacing so you use the same amount every time, it looks much cleaner then =) And remove unnecessary spacing. This is just details ofc!